integration_tests/contracts/package.json

```diff
@@ -0,0 +1,68 @@
+{
+  "name": "gravity-contracts",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "evm": "npx hardhat node",
+    "test": "npx hardhat test --network localhost",
+    "typechain": "npx hardhat typechain",
+    "compile-deployer": "npx tsc -p . ; pkg dist/contract-deployer.js --targets node10-linux-x64",
+    "evm_fork": "npx hardhat node --fork https://eth-mainnet.alchemyapi.io/v2/${ALCHEMY_ID} --fork-block-number 11780000",
+    "test_fork": "npx hardhat test --network localhost ./tests_mainnet_fork/*.ts"
+  },
+  "author": "Jehan Tremback",
```

Comment:
pre generated?

Reply:
https://github.com/PeggyJV/gravity-bridge/blob/main/solidity/package.json
Both the contracts and build system are copied from the gravity bridge repo directly.
A better approach may be to build it on the fly.

Reply:
https://github.com/crypto-org-chain/cronos/issues/38
Opened an issue, we could make it better the next time.